### PR TITLE
Update department model

### PR DESCRIPTION
**Pull request by matt-bernhardt**
#### Developer

- [x] All new ENV is documented in README
- [x] All new ENV has been added to Heroku Pipeline, Staging and Prod
- [x] ANDI or Wave has been run in accordance to
      [our guide](https://mitlibraries.github.io/guides/basics/a11y.html) and
      all issues introduced by these changes have been resolved or opened as new
      issues (link to those issues in the Pull Request details above)
- [ ] Stakeholder approval has been confirmed (or is not needed)

#### Code Reviewer

- [x] The commit message is clear and follows our guidelines
      (not just this pull request message)
- [x] There are appropriate tests covering any new functionality
- [x] The documentation has been updated or is unnecessary
- [x] The changes have been verified
- [x] New dependencies are appropriate or there were no changes

#### Requires database migrations?

YES

#### Includes new or updated dependencies?

NO

### CODE REVIEW

**Review by JPrevost — APPROVED** [review]

File: db/migrate/20210304142225_add_data_to_departments.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+class AddDataToDepartments < ActiveRecord::Migration[6.0]
+  def up
+    rename_column :departments, :name, :name_dw
+    add_column :departments, :code_dw, :string, null: false, default: ''
```
[the remark this answers is not in the document]
Comment (matt-bernhardt) [reply db/migrate/20210304142225_add_data_to_departments.rb:4]: This was one of the last changes that I made. We could probably go without both the null and default value, but I was trying to just go with `null: false` and that caused a problem.

I guess I could go either way.

**Comment by matt-bernhardt** [comment]
Tagging Helen for the data model review...

File: db/migrate/20210304142225_add_data_to_departments.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (JPrevost) [reply db/migrate/20210304142225_add_data_to_departments.rb:4]: I'm not blocking on this, mostly just asking. I think we end up in a fine place so :shipit: ?

**Comment by matt-bernhardt** [comment]
Helen approved this yesterday afternoon, and I apparently missed the notification - merging now...

**Pull request merged**